packages/terra-form-fieldset/src/Fieldset.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/jsx-boolean-value, jsx-a11y/label-has-for */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+import 'terra-base/lib/baseStyles';
+import styles from './Fieldset.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Children the Field contains.
+   */
+  children: PropTypes.node,
+  /**
+   * Help element to display with the input.
```

Comment:
Change this to just say 'Help Element to display', or 'Help element to display for the elements', as a fieldset is more than one input.